### PR TITLE
Added existing website content to sawtooth-website repo

### PR DESCRIPTION
**Pull request by vaporos**

### CODE REVIEW

**Comment by vaporos** [comment]
We probably need to determine which files _shouldn't_ be committed because they are not necessary. Also, should some of the third-party artifacts be excluded and pulled in with some build step?

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+| Name | GitHub | RocketChat |
+| --- | --- | --- |
+| Adam Ludvik | aludvik | adamludvik |
+| Andi Gunderson | agunde406 | agunde |
+| Anne Chenette | chenette | achenette |
+| Boyd Johnson | boydjohnson | boydjohnson |
+| Dan Middleton | dcmiddle | Dan |
+| Darian Plumb | dplumb94 | dplumb |
+| James Mitchell | jsmitchell | jsmitchell |
+| Kelly Olson | kelly\_ | ineffectualproperty |
```
Comment (ineffectualproperty) [inline MAINTAINERS.md]: Looks like Github and RocketChat got switched. Github is ineffectualproperty, RocketChat is kelly_

Comment (vaporos) [reply MAINTAINERS.md]: Fixed.

**Review by dcmiddle — APPROVED** [review]

**Review by agunde406 — APPROVED** [review]

**Review by rberg2 — APPROVED** [review]

**Comment by Sutrannu** [comment]
These files are not referenced in any other files.
```
htdocs/docs/imgs/favicon/apple-icon-180x180.png
htdocs/docs/imgs/favicon/apple-icon-precomposed.png
htdocs/docs/imgs/favicon/apple-icon.png
htdocs/docs/imgs/favicon/ms-icon-144x144.png
htdocs/docs/imgs/hero-bond-bg.png
htdocs/docs/imgs/hero-seafood-bg.png
htdocs/examples/img/favicon/apple-icon-180x180.png
htdocs/examples/img/favicon/apple-icon-precomposed.png
htdocs/examples/img/favicon/apple-icon.png
htdocs/examples/img/favicon/ms-icon-144x144.png
htdocs/examples/img/hero-bond-bg.png
htdocs/examples/img/hero-seafood-bg.png
htdocs/examples/img/overview-bond-bg.png
htdocs/examples/img/overview-seafood-bg.png
htdocs/examples/styles/sass/bond.scss
htdocs/examples/styles/sass/marketplace.scss
htdocs/examples/styles/sass/module/_bond.scss
htdocs/examples/styles/sass/module/_marketplace.scss
htdocs/examples/styles/sass/module/_seafood.scss
htdocs/examples/styles/sass/partial/_benefits.scss
htdocs/examples/styles/sass/partial/_challenges.scss
htdocs/examples/styles/sass/partial/_contact.scss
htdocs/examples/styles/sass/partial/_demos.scss
htdocs/examples/styles/sass/partial/_distributed-ledgers.scss
htdocs/examples/styles/sass/partial/_featured-resources.scss
htdocs/examples/styles/sass/partial/_footer.scss
htdocs/examples/styles/sass/partial/_many-use-cases.scss
htdocs/examples/styles/sass/partial/_overview.scss
htdocs/examples/styles/sass/partial/_solution.scss
htdocs/examples/styles/sass/seafood.scss
```

**Comment by vaporos** [comment]
Removed files @Sutrannu suggested for removal.

**Pull request merged**